t/014-output-unformatted.t

```diff
@@ -0,0 +1,29 @@
+use Test::Most 'die';
+use Test::More::UTF8;
+use Encode;
+use JSON::MaybeXS;
+
+my $tempfile;
+
+BEGIN {
+    use Path::Tiny;
+    $tempfile = Path::Tiny->tempfile;
+}
+
+use Log::Any '$log';
+use Log::Any::Adapter 'JSON', $tempfile->opena, format_message => 0;
+
+# last line logged
+sub last_line {
+    my $line = ($tempfile->lines({ chomp => 1 }))[-1];
+    return decode_json $line;
+}
+
+##
+subtest 'message with formatting codes' => sub {
+    $log->debug('%s and %d');
+    is last_line()->{message}, '%s and %d',                           'format codes ignored';
```

Comment:
We should have a test that if you pass the `without_formatting` arg, but *do* pass scalar values after the message, then those are included in `additional_messages`.
Something like:
```
$log->debug('%s and %d', 'should be', 'in','additional_messages');
my $line = last_line();
is $line->{message}, '%s and %d', '...';
is_deeply $line->{additional_messages}, ['should be','in','additonal_messages'],'...';
```